docs/modules/ROOT/nav.adoc

```diff
@@ -2,14 +2,14 @@
 * xref:overview.adoc[What is Hazelcast CLC?]
 * xref:get-started.adoc[Get Started]
 * Tutorials
-** xref:managing-viridian-clusters.adoc[Manage {hazelcast-cloud} Clusters]
+** xref:managing-cloud-clusters.adoc[Manage {hazelcast-cloud} Clusters]
```

Comment:
This is just a note:
I would assume changing the filenames will cause broken links outside of the documentation.
I don't have a solution for this, but I wonder whether it is possible to have redirects.

Reply:
Yes, redirects are in place in the affected files and the top-level build files. If someone is using a docs viridian link, it will resolve to the updated cloud link.

Reply:
That's cool!